documentation/plain_rules.md

```diff
@@ -104,16 +104,33 @@ Sends an email to the recipient set in the action parameters, with the body mail
 ```
 
 ### update attribute action
-Updates an specified attribute of the entity which caused the rule to be fired.
-This action does not use any value from the generated event  (except the id from the originating entity, but this is done transparently). The `parameters` field includes a field `name` for the name of the attribute to be changed and a `value` field with the value to be put in the attribute.
+Updates an specified attribute of the entity/event which caused the rule to be fired or another one.
+The `parameters` field must include a field `name` for the name of the attribute to be changed and a `value` field with
+the value to be put in that attribute.
 
+Optionally, an `id` attribute can be set in `parameters`. The value can be a literal value or a template in which an expression like `${X}`
+ will be substituted by the value of the attribute `X` from the entity firing the rule. So, if we want the entity with "id" `sensor1`
```

Comment:
Thus according with this description in `${X}`, the `X` is the name of an attribute in the entity firing the rule. However, I think that we defined in the IOTCORE-247 the following syntax:

* `${entityId}` to be replaced by the ID of the firing entity
* `${entityType}` to be replaced by the type of the firing entity
* `${attribute.X}` to be replaced by the value of the attribute `X` of the firing entity.

The documentation (and implementation) should be in that way. Or maybe I'm missing something?